Makefile

```diff
@@ -43,7 +43,7 @@ mini-init: setup pipenv-dev-install react-init autogen
 
 .PHONY: autogen
 # Generates files for frontend dev
-autogen: scssvars protobuf
+autogen: protobuf
```

Comment:
Since 'autogen' is just an alias for "protobuf", can we remove 'autogen' entirely from our codebase? AFAICT the only remaining usage is in init and mini-init above.

Reply:
Possible. I wasn't going to make the call cause I didn't want to disrupt other people's flow if they were using that.

Reply:
I vote do it, and put a 30 sec announcement at the next standup ;)